PointsOfInterest/aed.csv

```diff
@@ -0,0 +1,34 @@
+name,description,note,latitude,longitude
+,Between the male and female washrooms (across from the front desk),Columbia Lake Village (CLV),43.47082,-80.56244

```

Comment:
That being said... AED is supposed to be a universally accepted acronym: http://www.ilcor.org/news/news-archive/ilcor-presents-a-universal-aed-sign